packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/data_sources/local/users_local_data_source.dart

```diff
@@ -0,0 +1,24 @@
+{{> licence.dart }}
+
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+
+/// This class is used to save user profile to Secure Storage
+class UsersLocalDataSource {
```

Comment:
You can rephrase the beginning of the comment like `Data source used to ...`, as it sounds more indicative of what kind of class it is. Also, feel free to include comments for those methods.

Reply:
Done